sig-and-wg/wg-kyma-knative/README.md

```diff
@@ -0,0 +1,33 @@
+# Kyma Knative Working Group
+
+The purpose of this group is to:
+* Enable Knative serving in the Kyma.
```

Comment:
```suggestion
* Enable Knative serving in Kyma.
```

Reply:
@abbi-gaurav This comment uses the "suggestion" feature. All you have to do is merge my suggestion , no need to edit the file on your own.